src/db_init.h

```diff
@@ -26,7 +26,7 @@
  * is a major upgrade. In other words minor version upgrades permit downgrading
  * the server after the database was upgraded. */
 #define SCHEMA_VERSION_MAJOR 21
-#define SCHEMA_VERSION_MINOR 07
+#define SCHEMA_VERSION_MINOR 8
```

Comment:
I'm thinking the major would need to be upgraded, since the change doesn't seem backwards compatible. In the sense that if an older version of owntone is used with an upgraded database it might be adding rows where "source" would be set to NULL?